sticht/slack.py

```diff
@@ -87,9 +91,7 @@ async def get_slack_events():
         logging.error('Scribereader unavailable. Not tailing slack events.')
         return
 
-    host_and_port = scribereader.get_env_scribe_host(SCRIBE_ENV, True)
-    host = host_and_port['host']
-    port = host_and_port['port']
+    host, port = scribereader.get_tail_host_and_port(SCRIBE_ENV, True)
```

Comment:
O woops, forgot to mention also that the args should now be `get_tail_host_and_port(**SCRIBE_ENV)`, since the only three args are ecosystem, region and superregion in that order.